data/pins_sonata.xdc

```diff
@@ -177,32 +197,52 @@ set_property -dict { PACKAGE_PIN M16 IOSTANDARD LVCMOS33 } [get_ports rph_g3_scl
 # set_property PULLUP true [get_ports rph_g3_scl]
 ## rev 0.5+ have on-board pull ups
 
+## UART
+set_property -dict { PACKAGE_PIN K17 IOSTANDARD LVCMOS33 } [get_ports rph_txd0];
+set_property -dict { PACKAGE_PIN L18 IOSTANDARD LVCMOS33 } [get_ports rph_rxd0];
+
+## Other GPIO
+set_property -dict { PACKAGE_PIN N15 IOSTANDARD LVCMOS33 } [get_ports rph_g4];
```

Comment:
Perhaps not a priority, but it appears that a number of the R-Pi pins are not connected, particularly, eg. the SPI signals.

Reply:
They are if you look above this piece of code: SPI1, SPI0, I2C and UART are all connected.